### ISSUE TITLE
[feature request] command for adding tab to existing session

### ISSUE THREAD

**Issue by sim590** [qutebrowser/qutebrowser#3853]
I have some sessions which contain a lot of tabs. I would like to add one tab to them without opening them because memory may lack for e.g.

**Comment by jgkamat** [comment]
I'm not sure how this would work, you want to add a URL to a session file without actually loading it at all? So you would like `:session-add-url default https://example.com`? I suppose that could make sessions a way to collect URLs to store for later (in addition to a save/restore mechanism). You could probably write a userscript for this to collect the urls and write them out, but that would be a bit hacky...

**Comment by sim590** [comment]
> So you would like :session-add-url default https://example.com? I suppose that could make sessions a way to collect URLs to store for later (in addition to a save/restore mechanism).

Yes. It's already doing that job in a way since you can pass `--only-active-window` flag to `:session-save` which saves only the window you execute the command in. This is really handy when seperating different work on different windows and you want to postpone for instance some work for later.

You may have forgotten or dicovered one more tab you want to add to that session, but since then you have loaded a bunch more tabs and memory is already used at ~90% :wink:.

> You could probably write a userscript for this to collect the urls and write them out, but that would be a bit hacky...

Indeed, I see how it could be done, but that may as well be part of qutebrowser itself IMO.
